P2946-throws_nothing/P2946.md

```diff
@@ -106,7 +106,10 @@ safe framework for deciding whether a specific function should be declared
 
 The example below shows a subset of the `std::vector` interface. Note that only
 `size()`, which promises not to throw and has no preconditions, is declared
-`noexcept`, whereas the others each fail one or both of the Lakos-Rule tests
+`noexcept`, whereas the others each fail one or both of the Lakos Rule tests
+<!---
+    MG: be consistent regarding Lakos Rule hyphenation.
+-->
```

Comment:
Hyphenation is fine here since Lakos Rule is serving as an adj. But if we retain it here, the we have to make sure we've added it in every other place where it serves as an adj.